internal/store/db/redis/models/storedobject.go

```diff
@@ -0,0 +1,165 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+// models describes the data types that will be used when storing export data in Redis.
+package models
+
+import (
+	"encoding/json"
+	"github.com/edgexfoundry/app-functions-sdk-go/internal/store/contracts"
+)
+
+// StoredObject is the atomic and most abstract description of what is collected by the export store system.
+type StoredObject struct {
+	// ID uniquely identifies this StoredObject
+	ID string `json:"id"`
```

Comment:
@lenny-intel This is the approach you're looking for, right? Redis should have a model that does JSON unmarshalling? This should not be on the contract?

Reply:
Correct. Should mimic the Mongo model, but `json` instead of `bson`. Assuming Redis wants json... ;-)